### PR TITLE
[spec] Correct use of opdtype and stacktype

### PR DESCRIPTION
**Pull request by rossberg**
Fix editorial issue that came up in #1521.

### CODE REVIEW

File: document/core/valid/instructions.rst [hunk 1]
```diff
@@ -1424,7 +1424,7 @@ Control Instructions
   * For all :math:`l_i` in :math:`l^\ast`,
     and for each :ref:`operand type <syntax-opdtype>` :math:`t_j` in :math:`t^\ast` and corresponding type :math:`t'_{ij}` in :math:`C.\CLABELS[l_i]`, :math:`t_j` :ref:`matches <match-opdtype>` :math:`t'_{ij}`.
 
-* Then the instruction is valid with type :math:`[t_1^\ast~t^\ast~\I32] \to [t_2^\ast]`, for any sequences of :ref:`value types <syntax-valtype>` :math:`t_1^\ast` and :math:`t_2^\ast`.
+* Then the instruction is valid with type :math:`[t_1^\ast~t^\ast~\I32] \to [t_2^\ast]`, for any sequences of :ref:`operand types <syntax-opdtype>` :math:`t_1^\ast` and :math:`t_2^\ast`.
```
Comment (ia0) [inline document/core/valid/instructions.rst:1427]: I cannot comment on line 1420, but it states "result type". I believe it should be the equivalent using operand types. The bullet points below correctly mention operand types.

Comment (rossberg) [reply document/core/valid/instructions.rst:1427]: Right, fixed!

**Comment by ia0** [comment]
Thanks! (I don't want to approve because I'm not competent to do so, but this looks good to me regarding my issues in #1521.)

**Review by conrad-watt — APPROVED** [review]
LGTM

**Pull request merged**